app/views/users/sessions.py

```diff
@@ -141,6 +144,93 @@ def add_speaker_view(event_id, session_id):
         return redirect(url_for('.index_view', event_id=event_id))
 
 
+@event_sessions.route('/download_speakers_as_csv/', methods=('GET',))
+@belongs_to_event
+@can_access
+def download_speakers_as_csv(event_id):
+    sessions = DataGetter.get_sessions_by_event_id(event_id)
+    event = DataGetter.get_event(event_id)
+    data = "Speaker Name, Speaker Session(s) , Speaker Email, Speaker Status," \
+           " Speaker Mobile, Speaker Organisation, Speaker Position \n"
+
+    for session in sessions:
+        for speaker in session.speakers:
+            if speaker.name:
+                data += speaker.name+','
+            else:
+                data += ','
+            if session.title:
+                data += session.title+','
+            else:
+                data += ','
+            if speaker.email:
+                data += speaker.email+','
+            else:
+                data += ','
+            if speaker.sessions:
+                for sess in speaker.sessions:
+                    data += sess.state+','
+            else:
+                data += ','
+            if speaker.mobile:
+                data += speaker.mobile+','
+            else:
+                data += ','
+            if speaker.organisation:
+                data += speaker.organisation+','
+            else:
+                data += ','
+            if speaker.position:
+                data += speaker.position+','
+            else:
+                data += ','
+            data += '\n'
+
+    response = make_response(data)
+    response.headers['Content-Type'] = 'text/csv'
+    response.headers['Content-Disposition'] = \
+        'inline; filename=%s-Speakers.csv' % (re.sub(r"[^\w\s]", '', event.name).replace(" ", "_"))
+
+    return response
+
+
+@event_sessions.route('/download_sessions_as_csv/', methods=('GET',))
```

Comment:
@LuD1161 you don't need to mention `methods=('GET')`. By default, it anyway allows only 'GET'.